.github/workflows/CI.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     runs-on: ${{ matrix.os }}
```

Comment:
Also include py version.

Reply:
`runs-on` is only for machine types (refer [this](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idruns-on)). 

Python setup step is where python version matters and it also print the python version to ease out the debugging to figure which python version was used for that run.

```
      - name: Set up Python ${{ matrix.python-version }}
        uses: actions/setup-python@v2
        with:
          python-version: ${{ matrix.python-version }}
```

Not really sure what you meant by the comment. Let me know if you still see something missing.

Reply:
Commented at wrong line. This comment is applicable for the https://github.com/aws-greengrass/aws-greengrass-gdk-cli/blob/39b8bfd9d61e51f90eedb1e91fb23e633d1b5c60/.github/workflows/CI.yml#L25

Reply:
Sure .. now that makes sense :)